tests/samplers_tests/test_gp.py

```diff
@@ -28,7 +29,7 @@ def test_after_convergence(caplog: LogCaptureFixture) -> None:
     search_space = gp_search_space.SearchSpace(
         {"a": optuna.distributions.FloatDistribution(0.0, 1.0)}
     )
-    gpr = optuna._gp.gp.fit_kernel_params(
+    gpr = optuna_gp.fit_kernel_params(
```

Comment:
```suggestion
    gpr = gp.fit_kernel_params(
```
According to the suggestion above, this part needs to be updated.

https://github.com/optuna/optuna/blob/255fc3de0d64e7534bed24df2bfd0d4450bfb9a7/optuna/samplers/_gp/sampler.py#L259